CMakeLists.txt

```diff
@@ -325,13 +325,17 @@ add_subdirectory(3party/jansson)
 add_subdirectory(3party/liboauthcpp)
 add_subdirectory(3party/minizip)
 add_subdirectory(3party/opening_hours)
-add_subdirectory(3party/pugixml)
 add_subdirectory(3party/sdf_image)
 add_subdirectory(3party/stb_image)
 add_subdirectory(3party/succinct)
 add_subdirectory(3party/open-location-code)
 add_subdirectory(3party/vulkan_wrapper)
 
+find_package(pugixml 1.12)
```

Comment:
Ok, but now latest pugixml 1.12 doesn't have needed function set_value.
If we merged this PR now as is, we will break compilation for devs, who have system pugixml 1.12, no?

Reply:
Yeah I would recommend waiting with merging till either a new release of pugixml, or I can set this to 1.13 so it'll fail automatically till pugixml 1.13 is available.